Taarafo.Core/Brokers/Storages/StorageBroker.Groups.cs

```diff
@@ -3,13 +3,28 @@
 // FREE TO USE TO CONNECT THE WORLD
 // ---------------------------------------------------------------
 
+using System.Threading.Tasks;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
 using Taarafo.Core.Models.Groups;
 
 namespace Taarafo.Core.Brokers.Storages
 {
 	public partial class StorageBroker
 	{
 		public DbSet<Group> Groups { get; set; }	
+
+		public async ValueTask<Group> InsertGroupAsync(Group group)
+        {
+			using var broker = 
+				new StorageBroker(this.configuration);
+			
+			EntityEntry<Group> groupEntityEntry =
+				await broker.Groups.AddAsync(group);
+
+			await broker.SaveChangesAsync();
+
+			return groupEntityEntry.Entity;
+        }
```

Comment:
Indentations are not correct. If you copied this code from previous sessions I recommend typing it on your own line by line and try to understand what every line is trying to accomplish.